### PR TITLE
net-libs/qxmpp: add 1.5.5

### PR DESCRIPTION
**Pull request by CyberTailor**
OMEMO support is needed for Kaidan.

### CODE REVIEW

File: net-libs/libomemo-c/libomemo-c-0.5.0.ebuild
Comment (juippis) [inline net-libs/libomemo-c/libomemo-c-0.5.0.ebuild:13]: You should only include `~amd64` here, drop the keywords from the main package and request re-keywording through bugzilla once an update has been merged. 

Or we can use-mask omemo in x86 for the time being.

Also, no new packages straight to stable!

File: net-libs/libomemo-c/metadata.xml
Comment (juippis) [inline net-libs/libomemo-c/metadata.xml:4]: This is forbidden,
https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602

Would you like to maintain this library and become a co-/main maintainer with qxmpp? Otherwise we'd need @0xd34df00d to ack this PR and become a maintainer for this dep. To me it looks like these packages may need to be dropped to maintainer-needed if no one is maintaining them. Judging from the jump in versions with qxmpp, it looks rather neglected.

File: net-libs/libomemo-c/libomemo-c-0.5.0.ebuild
Comment (CyberTailor) [reply net-libs/libomemo-c/libomemo-c-0.5.0.ebuild]: Oops, really stupid mistake! Fixed.

File: net-libs/libomemo-c/metadata.xml
Comment (CyberTailor) [reply net-libs/libomemo-c/metadata.xml:4]: Yes, I can maintain these packages

Comment (CyberTailor) [reply net-libs/libomemo-c/metadata.xml]: Added to metadata

File: net-libs/libomemo-c/libomemo-c-0.5.0.ebuild
Comment (juippis) [inline net-libs/libomemo-c/libomemo-c-0.5.0.ebuild:12]: Could you look closer into this? For example, 
https://github.com/dino/libomemo-c/blob/omemo/src/vpool.c looks more like MIT or BSD (I never remember which one).

There may be other files with different licenses.

**Review by juippis — APPROVED** [review]
Let me know your opinion about the license thing, everything else looks ok!

File: net-libs/libomemo-c/libomemo-c-0.5.0.ebuild
Comment (CyberTailor) [reply net-libs/libomemo-c/libomemo-c-0.5.0.ebuild]: Indeed, found 3 more licenses. Fixed.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**
